### DISCUSSION TITLE
Laravel CRM Package

### DISCUSSION THREAD

**Discussion by saurav-webkul** [HassanZahirnia/laravel-package-ocean#71] [Package Suggestions]
Hello,

Could you add an opensource Larvel CRM Package; Krayin: https://github.com/krayin/laravel-crm

**Comment by HassanZahirnia** [comment]
Hey! Thanks for your package suggestion!

Is your package a commercial product?

Also if you want make it compatible with Laravel 10 I could have it added to the database.

**Comment by saurav-webkul** [comment]
Hi,

Yes, it can be used for commercial purposes as well. 

We are working on to upgrade to Laravel 10

**Reply by HassanZahirnia** [reply]
Let me know when you upgrade it to Laravel 10!

**Comment by saurav-webkul** [comment]
So after it is upgraded to Laravel 10, then you will include it?

**Reply by HassanZahirnia** [reply]
Yeah no problem as long as the package is actively maintained. But to be honest you should maintain it (of course if you're willing to) for the sake of your user base, not just for getting featured on this website ^_^
